cypress/e2e/GUI/testGui.cy.js

```diff
@@ -0,0 +1,44 @@
+/// <reference types="Cypress" />
+
+describe('Testando a aplicação EngageSphere', () => {
+
+    // Executa antes de cada teste
+    beforeEach(() => {
+      // Define uma data e hora fixa
+      const fixedDate = new Date(2024, 6, 10); // 10 de julho de 2024
+      cy.clock(fixedDate.getTime());
+      cy.visit('http://localhost:3000');
+    });
+  
+    // Bloco de testes
+    it('Testando o componente Header', () => {
+      cy.get('h1').should('have.text', 'EngageSphere');
```

Comment:
Veja uma alternativa mais enxuta, e que além disso, verifica pela visibilidade do elemento.

```suggestion
      cy.contains('h1', 'EngageSphere').should('be.visible');
```